applications/curamedon/README.md

```diff
@@ -0,0 +1,110 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2022 Inno Barn. All rights reserved.
+```
+# Curamedon
+
+## Introduction
+Curamedon is a ready-made web-based service that allows health practitioners’ offices to consult with patients via videoconferencing.
+This repository contains Helm Charts and instructions to help with Curamedon services deployment to Intel Smart Edge Open.
+For more information on Curamedon, please visit our website: [Curamedon](https://curamedon.com)
+
+## Prerequisites
+The application has been tested on the following software, which is also required:
+
+* Intel Smart Edge Open (release 21.09)
+* Kubernetes v1.20
+* Helm v3.1.2
+
+## Installation
+
+### Ingress Controller setup
+First thing to do is to set up ingress controller. Installation guide https://kubernetes.github.io/ingress-nginx/deploy.
+
+### Setup Cert Manager
+Next step is to set up cert manager. Use instructions from https://cert-manager.io/docs/installation/helm.
+
+Once cert manger is set up cluster issuers must be configured. Follow instruction from https://cert-manager.io/docs/configuration/acme/.
+It is expected that `letencrypt-prod` issuer is configured and ready to use.
+
+### Prepare StorageClass
+It is not recommended to use `hostPath` or `local` PV for production environment. The best way is to set up `NFS` or use some
+plugins to integrate with cloud storages (AWS, Azure etc.). At least one corresponding StorageClass that supports dynamic volume provisioning should be available by default.
+See details on https://kubernetes.io/docs/concepts/storage/storage-classes/
+
+### Create a namespace
+Run
+> kubectl create namespace telemedicine
```

Comment:
The "smartedge-apps" name space is available for the application. Please use this namespace instead of creating a new one.